admin_guide/managing_networking.adoc

```diff
@@ -559,7 +560,7 @@ metadata:
   labels:
     name: egress-multi
   annotations:
-    pod.network.openshift.io/assign-macvlan: "true"
+    pod.network.openshift.io/assign-macvlan: "true" <1>
```

Comment:
When I said "you don't have to document this much" in the Trello card, what I meant was, we don't expect very many users to use this feature. So I don't think you need to update *all* of the egress-router examples. Just the first one should be fine.